src/live_restore/live_restore_server.c

```diff
@@ -380,6 +374,25 @@ __wt_live_restore_server_create(WT_SESSION_IMPL *session, const char *cfg[])
 
     WT_CONNECTION_IMPL *conn = S2C(session);
 
+    /*
+     * We're currently using the default session. Create a real one for the clean up logic to
+     * perform checkpoints.
+     */
+    WT_SESSION_IMPL *checkpoint_session = NULL;
+
+    /*
+     * If background migration has already completed we don't need to start the background threads.
+     * Run the clean up logic regardless in case we've previously closed the connection at the exact
+     * time we finish migration, but before we call clean up.
```

Comment:
```suggestion
     * If background migration has already completed we don't need to start the background threads.
     * Run the clean up logic regardless in case we've previously closed the connection after
     * we finish migration, but before we call clean up.
```

Reply:
[6007cc3](https://github.com/wiredtiger/wiredtiger/pull/11798/commits/6007cc3b902f67f7a38e3bfef6f1696654066bd8) 👍